examples/bundle/pom.xml

```diff
@@ -364,13 +364,20 @@
             <artifactId>java-dataloader</artifactId>
             <version>2.2.3</version>
         </dependency>
+
         <dependency>
             <groupId>org.reactivestreams</groupId>
             <artifactId>reactive-streams</artifactId>
             <version>1.0.2</version>
         </dependency>
         
         <!-- Testing -->
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.api</artifactId>
+            <version>2.18.4</version>
```

Comment:
Isn't the version inherited from the parent POM?

Reply:
This dependency is not declared in the parent POM, it's only declared in the CIF Core Components bundle